bin/github-review.sh

```diff
@@ -0,0 +1,313 @@
+
+if [[ $cfh != $cfhCI ]]; then
+    echo "ERROR: CompatibilityFallbackHandler default address is not the same as the one created for the chain id" 1>&2
+    exit 1
+fi
+if [[ $cc != $ccCI ]]; then
+    echo "ERROR: CreateCall default address is not the same as the one created for the chain id" 1>&2
+    exit 1
+fi
+if [[ $msco != $mscoCI ]]; then
+    echo "ERROR: MultiSendCallOnly default address is not the same as the one created for the chain id" 1>&2
+    exit 1
+fi
+if [[ $ms != $msCI ]]; then
+    echo "ERROR: MultiSend default address is not the same as the one created for the chain id" 1>&2
+    exit 1
+fi
+if [[ $sml != $smlCI ]]; then
+    echo "ERROR: SignMessageLib default address is not the same as the one created for the chain id" 1>&2
+    exit 1
+fi
+if [[ $sta != $staCI ]]; then
+    echo "ERROR: SimulateTxAccessor default address is not the same as the one created for the chain id" 1>&2
+    exit 1
+fi
+
+if [[ $version == "1.3.0" ]]; then
+    if [[ $gsl2 != $gsl2CI ]]; then
+        echo "ERROR: GnosisSafeL2 default address is not the same as the one created for the chain id" 1>&2
+        exit 1
+    fi
+    if [[ $gs != $gsCI ]]; then
+        echo "ERROR: GnosisSafe default address is not the same as the one created for the chain id" 1>&2
+        exit 1
+    fi
+    if [[ $pf != $pfCI ]]; then
+        echo "ERROR: ProxyFactory default address is not the same as the one created for the chain id" 1>&2
+        exit 1
+    fi
+fi
+
+if [[ $version == "1.4.1" ]]; then
+    if [[ $sl2 != $sl2CI ]]; then
+        echo "ERROR: SafeL2 default address is not the same as the one created for the chain id" 1>&2
+        exit 1
+    fi
+    if [[ $spf != $spfCI ]]; then
+        echo "ERROR: SafeProxyFactory default address is not the same as the one created for the chain id" 1>&2
+        exit 1
+    fi
+    if [[ $s != $sCI ]]; then
+        echo "ERROR: Safe default address is not the same as the one created for the chain id" 1>&2
+        exit 1
+    fi
+fi
+
+echo "Default addresses are correct"
+
+# Code Hash of the default addresses (Chain ID: 1)
+# cast keccak $(cast code contractAddress --rpc-url https://rpc.ankr.com/eth)
+
+# v1.3.0
+if [[ $version == "1.3.0" ]]; then
+    cfhCH="0x03e69f7ce809e81687c69b19a7d7cca45b6d551ffdec73d9bb87178476de1abf"
+    ccCH="0x8155d988823a4f6f1bcbc76a64af8e510c4ce68819290d43cf24956bd24dee82"
+    gsl2CH="0x21842597390c4c6e3c1239e434a682b054bd9548eee5e9b1d6a4482731023c0f"
+    gsCH="0xbba688fbdb21ad2bb58bc320638b43d94e7d100f6f3ebaab0a4e4de6304b1c2e"
+    mscoCH="0xa9865ac2d9c7a1591619b188c4d88167b50df6cc0c5327fcbd1c8c75f7c066ad"
+    msCH="0x0208282bd262360d0320862c5ac70f375f5ed3b9d89a83a615b4d398415bdc83"
+    pfCH="0x337d7f54be11b6ed55fef7b667ea5488db53db8320a05d1146aa4bd169a39a9b"
+    smlCH="0x3ac65dea3cc9dd0d7b7b800f834e3d73415b4e944bb94555c3e4a08fb137e918"
+    staCH="0xb3fb9763869f2c09a2ac5a425d2dd6060bf7ef46b3899049d71a711e71e00f04"
```

Comment:
Idea for the future: Ideally these would be included in the json file

Reply:
(i approve after addressing Nick's comments) good job!

Reply:
> Ideally these would be included in the json file

Agree - you could read them with `jq` from a JSON file:

```jsonc
{
  "v1.3.0": {
    "GnosisSafe": "0xbba688fbdb21ad2bb58bc320638b43d94e7d100f6f3ebaab0a4e4de6304b1c2e",
    "GnosisSafeL2": "0x21842597390c4c6e3c1239e434a682b054bd9548eee5e9b1d6a4482731023c0f",
    // ...
  },
  "v1.4.1": {
    "Safe": "0x1fe2df852ba3299d6534ef416eefa406e56ced995bca886ab7a553e6d0c5e1c4",
    "SafeL2": "0xb1f926978a0f44a2c0ec8fe822418ae969bd8c3f18d61e5103100339894f81ff",
    // ...
  }
}
```

Can be read with:

```bash
jq -r --arg v $version --arg c $contract '.[$v].[$c]' contract-code-hashes.json 
```

Reply:
Updated 👍🏾